noria/src/data.rs

```diff
@@ -561,32 +562,25 @@ impl<'a> From<&'a str> for DataType {
     }
 }
 
-impl TryFrom<mysql::Value> for DataType {
+impl TryFrom<Value> for DataType {
```

Comment:
I think I prefer having this say `mysql_common::Value`. `Value` could be anything in a file like this.

Reply:
You could then add a `use` inside the `fn` for brevity.

Reply:
Or `use mysql_common::Value as MyValue`

Reply:
Done!